packages/fxa-settings/src/index.tsx

```diff
@@ -23,7 +25,7 @@ interface FlowQueryParams {
   uniqueUserId?: string;
 }
 
-// temporary until we can safely direct all users to all routes currently in content-server
+// Temporary query params
```

Comment:
Can we use a model with @bind for this. That'd alleviate the need for the cast on line 35.

Reply:
There are unfortunately a lot of places where we should be doing this. 😅 We do have FXA-7325 filed where we can clean them all up. This one in particular is getting removed next sprint.